app/controllers/schools/participants_controller.rb

```diff
@@ -160,14 +136,4 @@ def participant_mentor_form_params
   def email_used?(email)
     User.where(email:).where.not(id: @profile.user.id).any? || ParticipantIdentity.where(email:).where.not(user_id: @profile.user.id).any?
   end
-
-  def start_term_form_params
-    params.require(:participant_start_term_form).permit(:start_term)
-  rescue ActionController::ParameterMissing
-    nil
-  end
-
-  def build_start_term_form
-    ParticipantStartTermForm.new(start_term_form_params)
```

Comment:
Is this used anywhere else?  If not we can delete the file for the form to? 
https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/forms/participant_start_term_form.rb